### PR TITLE
Update Modal Ex

### PR DESCRIPTION
**Pull request by eXo5**
This is an example of a form inside a Modal.

### CODE REVIEW

**Comment by alextrastero** [comment]
Hey @eXo5 thanks for the PR, how about something a bit more simple like this:
```js
export default
<Modal
  header='Modal Header'
  modalOptions={{ dismissible: false, ready: () => { console.log('ready'); } }}
  trigger={<Button>MODAL</Button>}>
  <p>Lorem ipsum dolor sit amet</p>
</Modal>;
```

**Comment by eXo5** [comment]
What does PR stand for? And yeah, that's probably best - simpler is better I imagine.

**Comment by alextrastero** [comment]
Pull Request :)

On Aug 11, 2017 19:21, "Mike" <notifications@github.com> wrote:

What does PR stand for? And yeah, that's probably best - simpler is better
I imagine.

Sent from my iPhone

> On Aug 11, 2017, at 04:52, Alexander Odell <notifications@github.com>
wrote:
>
> Hey @eXo5 thanks for the PR, how about something a bit more simple like
this:
>
> export default
> <Modal
> header='Modal Header'
> modalOptions={{ dismissible: false, ready: () => { console.log('ready');
} }}
> trigger={<Button>MODAL</Button>}>
> <p>Lorem ipsum dolor sit amet</p>
> </Modal>;
> —
> You are receiving this because you were mentioned.

> Reply to this email directly, view it on GitHub, or mute the thread.
>

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/react-materialize/react-materialize/pull/327#issuecomment-321870594>,
or mute the thread

**Comment by eXo5** [comment]
Thanks :)

File: examples/Modal.js [hunk 1]
```diff
@@ -1,10 +1,47 @@
...
+     <h5>First Name</h5>
+      <Input type="text" id="firstName" value={this.state.firstName} onChange={this.handleChange}/>
+	   <h5>Last Name</h5>
+      <Input type="text" id="lastName" value={this.state.firstName} onChange={this.handleChange}/>
+	   <h5>email</h5>
+      <Input type="text" id="email" value={this.state.email} onChange={this.handleChange} />  
+    <Button id="submitButton" type="submit" className="mainBtn" onClick={this.handleSubmit}>Submit</Button>
+
+  <p>Press submit</p>
+</Modal>
```
Comment (isaiah) [inline examples/Modal.js]: This is syntactically not correct there is not closing parethesis and bracket. Please try it on your local. Simply go to the `docs/` folder and run `npm i && npm start`

**Pull request closed without merging**